roles/pulp_common/defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 
 pulp_cache_dir: '/var/lib/pulp/tmp'
+__pulp_artifact_dir: '/var/lib/pulp/artifact'
```

Comment:
This should be relative to `{{ pulp_user_home }}`, see:

https://github.com/pulp/pulp_installer/blob/master/roles/pulp_webserver/defaults/main.yml#L12

Reply:
done